Vagrantfile

```diff
@@ -30,10 +30,12 @@ Vagrant.configure("2") do |config|
         genhostid
         zpool create test -o cachefile=none -o multihost=on /dev/sdb
         zpool export test
-        curl https://sh.rustup.rs -sSf > rustup.sh
-        chmod +x rustup.sh
+        curl https://sh.rustup.rs -sSf > /home/vagrant/rustup.sh
+        chmod 777 rustup.sh
+        su vagrant
```

Comment:
I think you want the following lines inside the shell that is started with the `su`, yes with something like:

```
su vagrant <<EOF
  ./rustup.sh -y
  source $HOME/.cargo/env
EOF
```

Reply:
That works.